experimental/telemetry/tracker.go

```diff
@@ -0,0 +1,94 @@
+package telemetry
+
+import "github.com/mattermost/mattermost-plugin-api/experimental/bot/logger"
+
+// Tracker defines a telemetry tracker
+type Tracker interface {
+	// Track registers an event through the configured telemetry client
+	Track(event string, properties map[string]interface{})
```

Comment:
nit: apologies for the drive-by review, wondering if this should be `TrackEvent` for consistency with `TrackUserEvent`?